### ISSUE TITLE
A Piston Element not a block base

### ISSUE THREAD

**Issue by carl6005** [Team-MCToolkit/MCToolkit#133]
**Describe the feature**
You will be able to create pistons easily. 
**How would you implement it?**
I can see it working as an element where you would choose the textures for first the core piston block and then the head's texture. You would be able to adjust the length of it/how long it can reach/how long that middle piece between the core and head is.

Explain how you would like to be implemented in the UI (New mod element, element base, option, existing mod element, procedure block) and if you already tried to implement it on your own.
I would want to see it as a New mod element.

**Comment by Max094Reikeb** [comment]
A new mod element just for a piston is a lot, you should re-do your feature request for a block base...

**Comment by carl6005** [comment]
> A new mod element just for a piston is a lot, you should re-do your feature request for a block base...

I just don't see it working as a block base due to it consisting of 2 blocks

**Comment by titedog** [comment]
I don't think we should have this feature in the first place. It would be more efficient to have a tile-entity element because that would allow people to make a piston easily plus more tile-entity related things.

**Comment by carl6005** [comment]
> I don't think we should have this feature in the first place. It would be more efficient to have a tile-entity element because that would allow people to make a piston easily plus more tile-entity related things.

_ think that could work as well_

**Comment by Max094Reikeb** [comment]
I don't think a whole mod element would be needed for Pistons. I don't see the usage of it as well, but if you want to make your own piston, it might be possible to make one with procedures and block mod element...

**Comment by Goldorion** [comment]
A new mod element is not needed and won't be added as a new mod element. If this feature has to be added, it will be added as a block base.
